src/fe-text/irssi.c

```diff
@@ -74,11 +75,20 @@ void mainwindow_activity_deinit(void);
 void mainwindows_layout_init(void);
 void mainwindows_layout_deinit(void);
 
+#ifdef HAVE_SECCOMP
```

Comment:
@ahf was actually the one to tell me to do it that way. Previously I had done it something like like so:
```c
#ifdef HAVE_SECCOMP
if (/*stuff that checks if --sandbox is passed on the command line*/)
    int sandbox = 1;
#else
const int sandbox = 0;
#endif
```

But was told instead to change it to be in the `if (sandbox)` statement in `irssi.c`, so the `--sandbox` argument would always be available, but would display an error if used on a build without sandbox support.